### PR TITLE
add snapshot test for syntax highlight grammar

### PR DESCRIPTION
**Pull request by jasonlyu123**
the test setup is adapted from [vscode-ng-language-service](https://github.com/angular/vscode-ng-language-service). Might also have to add some edge cases we fixed before.

### CODE REVIEW

File: packages/svelte-vscode/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "build": "npm run build:ts && npm run build:grammar",
         "vscode:prepublish": "npm run build && npm prune --production",
         "watch": "npm run build:grammar && tsc -w -p ./",
-        "test": "echo 'NOOP'"
+        "test": "npm run build:grammar && node test/grammar/test.js"
```
Comment (jasonlyu123) [inline packages/svelte-vscode/package.json:12]: The reason for this `test.js` script is that we can just read the `dummy` directory to build the argument list instead of having to write a big long list in the `package.json`

**Review by dummdidumm — APPROVED** [review]
Awesome :tada: This will help us greatly to adjust the grammar in the future and not having to worry about regressions - so much better than having a manual look at all the colors.
I think the tests cover the syntax pretty well, we can always add more if we run into something.
